collect_app/src/test/java/org/odk/collect/android/preferences/stores/AdminPreferencesDataStoreTest.kt

```diff
@@ -0,0 +1,96 @@
+package org.odk.collect.android.preferences.stores
+
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.odk.collect.android.preferences.PreferencesDataSource
+import org.odk.collect.android.preferences.PreferencesDataSourceProvider
+
+class AdminPreferencesDataStoreTest {
+    private lateinit var adminPreferencesDataStore: AdminPreferencesDataStore
+    private lateinit var preferencesDataSourceProvider: PreferencesDataSourceProvider
+    private lateinit var adminPreferences: PreferencesDataSource
+
+    @Before
+    fun setup() {
+        preferencesDataSourceProvider = mock(PreferencesDataSourceProvider::class.java)
+        adminPreferences = mock(PreferencesDataSource::class.java)
+        `when`(preferencesDataSourceProvider.getAdminPreferences()).thenReturn(adminPreferences)
+        adminPreferencesDataStore = AdminPreferencesDataStore(preferencesDataSourceProvider)
+    }
+
+    @Test
+    fun `When putString() called, should save() be called on adminPreferences`() {
+        adminPreferencesDataStore.putString("Key", "value")
+        verify(adminPreferences, times(1)).save("Key", "value")
```

Comment:
Don't think you need the `times` in these tests.

Reply:
Right, Fixed.